File: Leetcode-Easy/58-LengthOfLastWord/Solution.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count = 0;
        for(int i = s.length()-1 ; i>=0 ; i--){
            if(s.charAt(i) != ' '){
                count++;
            }
            else{
                if(count>0){
                    return count;
                }
            }
        }
        return count;
    }
}

File: Leetcode-Easy/58-LengthofLastWord/Solution.c
int lengthOfLastWord(char * s){
    int i;
    int k=0;
    for (i = 0; s[i] != '\0'; ++i);

    for(int j=i; j>=0; j--){
        if(s[j]!='\0'&&s[j]!=' '){
            k++;
        }

        else if(s[j]==' ' && k>0){
            break;
        }
    }
    return k;
}
